src/test/kotlin/com/exactpro/th2/common/schema/message/impl/rabbitmq/connection/TestConnectionManager.kt

```diff
@@ -707,6 +709,190 @@ class TestConnectionManager {
             }
     }
 
+    @Test
```

Comment:
I think you might consider using TestFactory for this. It allows to creation of multiple tests from a single method. Not sure if it fits into the requirements you need but you might take a look and decide for yourself

Reply:
I wrote this method to allow to  execute multiple tests in parallel because of long delay needed in this test.